embed_extractor.py

```diff
@@ -14,6 +15,7 @@ def load_video_from_url(in_url):
         reqObj = http.send_request(in_url)
         page_content = reqObj.text
         url = reqObj.url
+        time.sleep(1)
```

Comment:
Thanks once more :)
Please just move sleep into 9anime_direct
This will ensure we wont sleep on openload 👍